src/main/resources/application.properties

```diff
@@ -46,6 +46,7 @@ waitlist.resend.fixed.rate: 15000
 failed.notifications.collection.name: failed_notifications
 failed.notifications.collection.ttl: 600
 notification.retry: 3
+notification.httpRequest.timeout:3000
```

Comment:
Check the comment from the other PR about following the the Spring Properties naming conventions:
https://github.com/eiffel-community/eiffel-intelligence/pull/503#discussion_r681574470

Reply:
Have modified the property accordingly but now this new configurable parameter is removed..We are not using this

Reply:
@tobiasake Have had time to check this again?

Reply:
Where is the change? Has this line in application.config been removed?